action/protocol/rewarding/reward.go

```diff
@@ -157,7 +157,7 @@ func (p *Protocol) GrantEpochReward(
 		exemptAddrs[addr.String()] = nil
 	}
 
-	var uqd map[string]bool
+	uqd := make(map[string]bool)
```

Comment:
i think it is ok to define a nil map here.
we can make the map only in the else statement

Reply:
ok! changed in latest commit